File: LeetCode/Java/0034_Find_First_and_Last_Position_of_Element_in_Sorted_Array.java
class Solution {
    public int[] searchRange(int[] arr, int target) {
        int[] positions = {-1, -1};
        
        for (int i = 0; i < arr.length; i++)
            if (arr[i] == target) {
                positions[0] = i;
                break;
            }
        
        if (positions[0] == -1)
            return positions;
        
        for (int i = arr.length - 1; i >= 0; i--) 
            if (arr[i] == target) {
                positions[1] = i;
                break;
            }
        
        return positions;
    }
}

File: LeetCode/Python/0034_Find_First_and_Last_Position_of_Element_in_Sorted_Array.py
class Solution:
    def searchRange(self, arr: List[int], target: int) -> List[int]:
        positions = [-1, -1]
        
        for i in range(len(arr)):
            if arr[i] == target:
                positions[0] = i
                break
        
        if positions[0] == -1:
            return positions
        
        for i in range(len(arr) - 1, -1, -1):
            if arr[i] == target:
                positions[1] = i
                return positions
